.github/workflows/release.yml

```diff
@@ -26,81 +26,81 @@ jobs:
     name: "Perform any build or bundling steps, as necessary."
     uses: ./.github/workflows/build.yml
 
-  # ssdlc:
-  #   needs: [release_please, build]
-  #   permissions:
-  #     # required for all workflows
-  #     security-events: write
-  #     id-token: write
-  #     contents: write
-  #   environment: release
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - uses: actions/checkout@v4
+  ssdlc:
+    needs: [release_please, build]
+    permissions:
+      # required for all workflows
+      security-events: write
+      id-token: write
+      contents: write
+    environment: release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
 
-  #     - name: Install Node and dependencies
-  #       uses: mongodb-labs/drivers-github-tools/node/setup@v2
-  #       with:
-  #         ignore_install_scripts: true
+      - name: Install Node and dependencies
+        uses: mongodb-labs/drivers-github-tools/node/setup@v2
+        with:
+          ignore_install_scripts: true
 
-  #     - name: Load version and package info
-  #       uses: mongodb-labs/drivers-github-tools/node/get_version_info@v2
-  #       with:
-  #         npm_package_name: mongodb-client-encryption
+      - name: Load version and package info
+        uses: mongodb-labs/drivers-github-tools/node/get_version_info@v2
+        with:
+          npm_package_name: "@mongodb-js/zstd"
 
-  #     - name: actions/compress_sign_and_upload
-  #       uses: mongodb-labs/drivers-github-tools/node/sign_node_package@v2
-  #       with:
-  #         aws_role_arn: ${{ secrets.AWS_ROLE_ARN }}
-  #         aws_region_name: us-east-1
-  #         aws_secret_id: ${{ secrets.AWS_SECRET_ID }}
-  #         npm_package_name: mongodb-client-encryption
-  #         dry_run: ${{ needs.release_please.outputs.release_created == '' }}
-  #         sign_native: true
+      - name: actions/compress_sign_and_upload
+        uses: mongodb-labs/drivers-github-tools/node/sign_node_package@v2
+        with:
+          aws_role_arn: ${{ secrets.AWS_ROLE_ARN }}
+          aws_region_name: us-east-1
+          aws_secret_id: ${{ secrets.AWS_SECRET_ID }}
+          npm_package_name: "@mongodb-js/zstd"
+          dry_run: ${{ needs.release_please.outputs.release_created == '' }}
+          sign_native: true
 
-  #     - name: Copy sbom file to release assets
-  #       shell: bash
-  #       if: ${{ 'mongodb-client-encryption-6.1' == '' }}
-  #       run: cp sbom.json ${{ env.S3_ASSETS }}/sbom.json
+      - name: Copy sbom file to release assets
+        shell: bash
+        if: ${{ 'node-zstd' == '' }}
```

Comment:
What is this condition for? it is never true 🤔

Reply:
We use the same template to generate the release action for packages with 3rd party dependencies and without.  This step is only relevant for packages with 3rd party dependencies.  

If you look in the driver, the opposite condition is true.